zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -214,9 +214,7 @@ class Handlers(queryExtractor: QueryExtractor) {
       for (traces <- tracesCall) yield {
         val (annotations, binaryAnnotations) = queryExtractor.getAnnotations(req)
         val data = Map(
-          "traces" -> traceSummaryToMustache(serviceName, traces),
-          "annotations" -> annotations,
-          "binaryAnnotations" -> binaryAnnotations
```

Comment:
It didn't look like annotations and binaryAnnotations were in use, in index.mustache.